biome.json

```diff
@@ -0,0 +1,6 @@
+{
+    "$schema": "https://biomejs.dev/schemas/1.6.3/schema.json",
+    "formatter": {
+      "indentStyle": "space"
+    }
+  }
```

Comment:
Add a newline at the end of files. You can configure VSCode to do this for you! 

See `files.insertFinalNewLine`: https://github.com/microsoft/vscode/blob/3648e8f381427e8b8352f8bf62e26b81cb3b7035/.vscode/settings.json#L109

Reply:
I thought biome fmt would automtically format their config file 😅. Fixed.